### DISCUSSION TITLE
Welcome to AbyssEngine Discussions!

### DISCUSSION THREAD

**Discussion by essial** [AbyssEngine/AbyssEngineOld#148] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Temasdin** [comment]
Hi!
I'm a game developer who's deeply in love with ARPGs, especially Diablo 2.

Can you give a bit of a historical overview: why did development start with Go, then switch to C++, and finally end up in C? Was it for the sake of authenticity?
